samples/Mvc.FileUpload/MultipartRequestHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.IO;
+using Microsoft.Net.Http.Headers;
+
+namespace Mvc.FileUpload
+{
+    public static class MultipartRequestHelper
```

Comment:
Ah it wasn't showing your comments when I was looking at it. Yeah we need to improve this, this isn't practical.